app/controllers/journals_controller.rb

```diff
@@ -1,7 +1,14 @@
 class JournalsController < ApplicationController
-
   def index
```

Comment:
do like this:
1. return a redirect to journal path with student grade id if the current user is a student
2. render journal/list_of_grades template if the current user is a teacher
3. render journal/list_of_students template if the current user is a parent